batcher/aligned-sdk/src/core/types.rs

```diff
@@ -328,6 +344,56 @@ pub enum ValidityResponseMessage {
     InvalidPaymentServiceAddress(Address, Address),
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum ProofInvalidReason {
+    Unknown,
+    MissingVerificationData,
```

Comment:
really like these enums. Given the context where this one was used, I think that instead of `VerificationData` it was meant to say `VerificationKey`